### ISSUE TITLE
expose `stream` method on conn

### ISSUE THREAD

**Issue by bsnacks000** [cunybpl/aiodal#27]
@tinnaing347 We use `transaction.execute` 9/10, but for certain client applications i'm using with anyio it is cleaner to stream data in move it through different producer consumer pipelines. 

The `connection` object exposes the `stream` method for just that. In an app i'm working on I'm doing this: 
```python 

stream = await transaction._conn.stream(stmt)
async for meter in stream:
    yield meter.meterid

```

It would be nice to just expose this via the proxy as well as its underlying signature in our TransactionManager object

**Comment by bsnacks000** [comment]
At this point maybe its best to just expose `_conn` directly with simple `@property` and just start using that... the transaction manager is more a container anyway. 

We can still use `execute` `commit` and `rollback` methods for convenience but exposing `_conn` directly will avoid us having to keep patching this object the more we expose

**Comment by tinnaing347** [comment]
closed by #29
